### ISSUE TITLE
Kicad project files

### ISSUE THREAD

**Issue by heyitscassio** [Mposiblee/Zireael#2]
Hello! That's a pretty nice keyboard.

It's on your interest to push the kicad project files to this repo? I'm interested in making another project based on yours (Keeping your credits of course).

**Comment by Mposiblee** [comment]
Yes, I can post it. If it's convenient, you can write to me in telegram, I'll send the files, it'll be faster
